src/panel/direction/road_map_panel.js

```diff
@@ -37,10 +37,24 @@ export default class RoadMapPanel {
 
   toggleRoute(i) {
     fire('toggle_route', i)
+
+    this.routes.forEach((route, index) => {
+      route.isActive = false
+      if(!Device.isMobile()){
+        this.panel.removeClassName(0, `#itinerary_leg_${index}`, 'itinerary_leg--active')
+      }
+    });
+
+    this.routes[i].isActive = true
```

Comment:
i'm not sure a loop is necessary. You should keep the current active route and toggle it status then toggle the status of the new current route and save it in the current route.